python/sglang/srt/layers/attention/triton_ops/decode_attention.py

```diff
@@ -406,6 +406,10 @@ def _decode_grouped_att_m_fwd(
     Lk = k_buffer.shape[-1]
     Lv = v_buffer.shape[-1]
 
+    # [TODO] work around shmem limit on MI3xx
+    if is_hip_ and Lk >= 576:
```

Comment:
If this workaround is used here, can the modified `moe_align_kernel` be omitted on AMD?

Reply:
@zhyncs oh I thought I committed this already.
This is needed for large Lk here on AMD.
The change to use `moe_align_kernel` under `sgl-kernel` is still needed, the two are not rooted the same.